### DISCUSSION TITLE
Does the ORM Model have to define each column of the table？

### DISCUSSION THREAD

**Discussion by cnzoujun** [adonisjs/core#4140] [Help] [answered]
Do each column of the table need to be defined in the model?Laravel, no need。
 If this is necessary, can there be tools to help me automatically complete it when there are many columns in the table.
such as [sequelize-auto](https://github.com/sequelize/sequelize-auto)

**Comment by RomainLanz** [comment, marked as answer]
Hey @cnzoujun! 👋🏻 

GitHub Copilot will do it for you if you also have the migration or use a common term/name.
We don't have any tools for the moment to automatically adds column.

And we know you don't need it with Laravel. That is a strong pain while using Laravel since you have no completion... 😉

**Reply by cnzoujun** [reply]
Thank you very much for your answer！

**Reply by cnzoujun** [reply]
I found it in the discussion, what I need is this：Creating models from existing database #1356
